contracts/Staking.sol

```diff
@@ -35,7 +36,9 @@ contract Staking {
         _;
     }
 
-    constructor() {}
+    constructor(uint32 maxNumValidators) {
+        _maximumNumValidators = maxNumValidators;
```

Comment:
For now, we don't use constructor for pre-deploy. I think you don't need to set _maximumValidators here

Reply:
Thanks ! I'll update that

Reply:
As this is the only way to provide a value for testing purposes and considering the fact that the constructor is not used for pre-deploy, we'll leave it for now.